src/bundle/Controller/LanguageController.php

```diff
@@ -107,8 +107,9 @@ public function listAction(Request $request): Response
         return $this->render('@EzPlatformAdminUi/admin/language/list.html.twig', [
             'pager' => $pagerfanta,
             'form_languages_delete' => $deleteLanguagesForm->createView(),
-            'canEdit' => $this->isGranted(new Attribute('content', 'translations')),
-            'canAssign' => $this->isGranted(new Attribute('content', 'translations')),
+            'can_edit' => $this->isGranted(new Attribute('content', 'translations')),
+            'can_delete' => $this->isGranted(new Attribute('content', 'translations')),
+            'can_create' => $this->isGranted(new Attribute('content', 'translations')),
```

Comment:
Why do we need 3 variables containing the same thing?

Reply:
Indeed something is wrong here. I'm also not sure how `$this->isGranted()` is working, I'd rather use `PermissionResolver::hasAccess` for that. But `isGranted()` might be okay, just never looked what it does internally.